src/Donations/index.tsx

```diff
@@ -262,13 +282,45 @@ function DonationInfo() {
                   style={{ marginTop: "10px" }}
                 >
                   {projectDetails.name}
+                  {isApproved && (
+                    <div className="d-inline" style={{ marginLeft: "10px" }}>
+                      <Typography
+                        aria-owns={open ? "mouse-over-popover" : undefined}
+                        aria-haspopup="true"
+                        onMouseEnter={handlePopoverOpen}
+                        onMouseLeave={handlePopoverClose}
+                      >
+                        <VerifiedIcon sx={{ color: "#fff" }} />
+                      </Typography>
+                      <Popover
+                        id="mouse-over-popover"
+                        className="verified-icon-popup"
+                        open={open}
+                        anchorEl={anchorEl}
+                        anchorOrigin={{
+                          vertical: "bottom",
+                          horizontal: "left",
+                        }}
+                        transformOrigin={{
+                          vertical: "top",
+                          horizontal: isMobile ? "center" : "left",
+                        }}
+                        onClose={handlePopoverClose}
+                        disableRestoreFocus
+                        marginThreshold={0}
+                      >
+                        <Typography>{t("verifiedIconInfo")}</Typography>
+                      </Popover>
+                    </div>
+                  )}
                 </a>
               ) : (
                 <h1
                   className="title-text text-white"
                   style={{ marginTop: "10px" }}
                 >
                   {projectDetails.name ? projectDetails.name : ""}
+                  {projectDetails.name && <VerifiedIcon />}
```

Comment:
If project is not a "tree" project, the verified icon always shows up. This includes funding, conservation, bouquet projects and planetcash. 

1. Is this behaviour correct?
2. If correct, the alignment needs to be adjusted - it differs from the alignment seen for tree projects.

Reply:
Resolved - made a minor adjustment to allow conservation projects to function in the same way as tree projects i.e. with a linked name.